tests/admin_views/tests.py

```diff
@@ -1809,6 +1810,44 @@ def test_history_view(self):
 
             self.client.get(reverse('admin:logout'))
 
+    def test_history_view_changed_fields_message(self):
+        """
+        Test the history_view change message contains indication of the changed fields as their form label
+        """
+        login_url = '%s?next=%s' % (reverse('admin:login'), reverse('admin:index'))
```

Comment:
I think you could simplify this a bit by using `self.client.login(self.super_login)` and the ORM to create the initial objects instead of the add view.